### PR TITLE
Kryonite Labs website MVP

### PR DESCRIPTION
**Pull request by lusu007**
## Proposed Changes

- add docusaurus template
- add custom homepage
- add github actions
- add pr and issue templates
- add dependabot
- add first blog post
- add start of documentation

### CODE REVIEW

**Comment by lusu007** [comment]
Currently waiting for a logo. 😄

**Comment by lusu007** [comment]
Currently blocked by https://github.com/kryonitelabs/docs/pull/1

File: .github/workflows/documentation.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+name: Test and Release 🏗️
+on:
+  pull_request:
+    branches: [main]
+  push:
+    branches: [main]
+jobs:
+  checks:
+    if: github.event_name != 'push'
```
Comment (ItsKev) [inline .github/workflows/documentation.yml:9]: Wouldn't it be better to create two workflows here? There isn't really a benefit doing the two checks in one workflow right?

File: .github/workflows/documentation.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      - name: Set up SSH deployment ⚙️
+        uses: webfactory/ssh-agent@v0.5.3
+        with:
+          ssh-private-key: ${{ secrets.GH_PAGES_DEPLOY }}
+      - name: Release to GitHub Pages 🚀
+        env:
+          USE_SSH: true
+          GIT_USER: git
+        run: |
+          git config --global user.email "lukas.jost+kryonitelabs@outlook.com"
```
Comment (ItsKev) [inline .github/workflows/documentation.yml:47]: Shouldn't this use the bots email?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+```
+$ yarn build
+```
+
+This command generates static content into the `build` directory and can be served using any static contents hosting service.
+
+### Deployment
+
+```
+$ GIT_USER=<Your GitHub username> USE_SSH=true yarn deploy
```
Comment (ItsKev) [inline README.md:30]: Is this a multi or single line command?

File: blog/2021-09-16-what-is-kryonite-labs.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+Over the past few years, we have gained a lot of experience in the Minecraft development scene. One of the big problems we have constantly noticed is that many networks and developers are still working with old technology stacks. Unfortunately, many projects are not maintained or are no longer developed at all. With [Kryonite Labs](https://github.com/kryonitelabs/), we want to create a space for new projects with new possibilities for Minecraft.
+
+**We want to offer our users the most modern and best user experience.**
+
+## 🚧 Our projects
+
+### 🌨️ Kryocloud
+
+The main project of [Kryonite Labs](https://github.com/kryonitelabs/) is our [Kryocloud](./docs/kryocloud/concept). Actually, we wanted to remove the word "cloud" from the name, as this software is not a cloud at all, but an orchestrator. However, the name "cloud" for that kind of software has become widespread in the scene... 🤷‍♂️
```
Comment (ItsKev) [inline blog/2021-09-16-what-is-kryonite-labs.md:18]:
```suggestion
The main project of [Kryonite Labs](https://github.com/kryonitelabs/) is [Kryocloud](./docs/kryocloud/concept). We didn't want to include the word "cloud" in the project's name, as this software is not a cloud, but an orchestrator. However, the name "cloud" for that kind of software has become widespread in the scene... 🤷‍♂️
```

**Review by ItsKev — CHANGES_REQUESTED** [review]

File: .github/workflows/documentation.yml (hunk 2 above)
Comment (lusu007) [reply .github/workflows/documentation.yml:47]: Yeah... It's surely an artifact of the time when we didn't have a bot account yet 🤣

File: .github/workflows/documentation.yml (hunk 1 above)
Comment (lusu007) [reply .github/workflows/documentation.yml:9]: Good point. I will change it.

File: README.md (hunk 3 above)
Comment (lusu007) [reply README.md:30]: The complete README.md file is generated by Docusaurus itself. I think the question is wether we change the README.md at all @ItsKev?

**Review by ItsKev — APPROVED** [review]

**Pull request merged**